Fix frontend build

```
$ npm run build
> frontend@0.0.0 build
> tsc -b && vite build

src/App.tsx:6:42 - error TS2339: Property 'authFetch' does not exist on type 'AuthContextType'.
6   const { isAuthenticated, user, logout, authFetch } = useAuth()
                                           ~~~~~~~~~
src/App.tsx:12:16 - error TS7006: Parameter 'res' implicitly has an 'any' type.
12         .then((res) => {
                  ~~~
src/App.tsx:16:16 - error TS7006: Parameter 'data' implicitly has an 'any' type.
16         .then((data) => setCount(data.count))
                  ~~~~
src/App.tsx:17:15 - error TS7006: Parameter 'data' implicitly has an 'any' type.
17         .then(data => setCount(data.count))
                 ~~~~
src/App.tsx:18:24 - error TS7006: Parameter 'err' implicitly has an 'any' type.
18                 .catch(err => {
                          ~~~
src/contexts/AuthContext.tsx:93:7 - error TS2353: Object literal may only specify known properties, and 'authFetch' does not exist in type 'AuthContextType'.
93       authFetch
         ~~~~~~~~~
  node_modules/@types/react/index.d.ts:535:9
    535         value: T;
                ~~~~~
    The expected type comes from property 'value' which is declared here on type 'IntrinsicAttributes & ProviderProps<AuthContextType | null>'

Found 6 errors.
```

--- a/frontend/src/App.tsx
+++ b/frontend/src/App.tsx
@@ -9,20 +9,19 @@ function App() {
   useEffect(() => {
     if (isAuthenticated) {
       authFetch("/api")
-        .then((res) => {
+        .then((res: Response) => {
           if (!res.ok) throw new Error(`HTTP error! Status: ${res.status}`)
           return res.json()
         })
-        .then((data) => setCount(data.count))
-        .then(data => setCount(data.count))
-                .catch(err => {
-                  console.error("Failed to fetch count:", err)
-                  if (err.message === 'Session expired') {
-                    logout()
-                  }
-                })
-            }
-  }, [isAuthenticated])
+        .then((data: { count: number }) => setCount(data.count))
+        .catch((err: Error) => {
+          console.error("Failed to fetch count:", err)
+          if (err.message === 'Session expired') {
+            logout()
+          }
+        })
+    }
+  }, [isAuthenticated, authFetch, logout])
 
   if (!isAuthenticated) {
     return <Login />

--- a/frontend/src/contexts/AuthContext.tsx
+++ b/frontend/src/contexts/AuthContext.tsx
@@ -15,6 +15,7 @@ interface AuthContextType {
   login: (username: string, password: string) => Promise<void>
   logout: () => void
   isAuthenticated: boolean
+  authFetch: (url: string, options?: RequestInit) => Promise<Response>
 }
 
 const AuthContext = createContext<AuthContextType | null>(null)

